plugins/lsp/language-servers/default.nix

```diff
@@ -538,6 +538,11 @@ let
       name = "texlab";
       description = "texlab language server for LaTeX";
     }
+    {
+      name = "tflint";
+      description = "tflint, a terraform linter";
+      package = pkgs.tflint;
+    }
```

Comment:
```suggestion
      description = "tflint, a terraform linter";
    }
```

Reply:
Forgot that `mkLsp` would handle that :+1: